communication/slack-config/channels.yaml

```diff
@@ -3,6 +3,8 @@
 
 channels:
   - name: airflow-operator
+  - name: aks-engine-users
+  - name: aks-engine-devs
```

Comment:
I don't think it's a hard requirement but it would be more consistent if we change this to the `-dev` suffix. cc @craiglpeters

Reply:
Makes sense, thanks @idealhack.